### PR TITLE
implement the sort with swaps plugin

### PR DESCRIPTION
**Pull request by fayllkw**
implement the sort with swaps plugin

### CODE REVIEW

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+
+import java.util.Random;
+
+public class SortWithSwapsPlugin implements GamePlugin {
+    private int ONE = 1;
+    private int TEN = 10;
+
+    // Describes whether the current move is over
+    public Boolean move = false;
```
Comment (mengqiuteng) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java:13]: could have used primitive type

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
+package edu.cmu.cs.cs214.rec10.plugin;
+
+import edu.cmu.cs.cs214.rec10.framework.core.GameFramework;
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+
+import java.util.Random;
+
+public class SortWithSwapsPlugin implements GamePlugin {
+    private int ONE = 1;
```
Comment (mengqiuteng) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java:9]: should be static final.

**Review by mengqiuteng — CHANGES_REQUESTED** [review]
We should prefer primitive types, use private accessors and use static final for constants.

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java (hunk 2 above)
Comment (fayllkw) [reply src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java]: Fixed in the newest commit.

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java (hunk 1 above)
Comment (fayllkw) [reply src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java]: Fixed in the newest commit.

**Review by mengqiuteng — APPROVED** [review]

**Pull request merged**